java/src/test/java/ai/rapids/cudf/ColumnViewNonEmptyNullsTest.java

```diff
@@ -41,11 +42,10 @@ void testAndNullReconfigureNulls() {
          ColumnVector intResult = v1.mergeAndSetValidity(BinaryOp.BITWISE_AND, v0);
          ColumnVector v2 = ColumnVector.fromStrings("0", "100", "1", "2", "MIN_VALUE", "3");
          ColumnVector stringResult = v2.mergeAndSetValidity(BinaryOp.BITWISE_AND, v0, v1);
-         ColumnVector stringExpected = ColumnVector.fromStrings("0", "100", null, null, "MIN_VALUE", null);
-         ColumnVector noMaskResult = v2.mergeAndSetValidity(BinaryOp.BITWISE_AND)) {
+         ColumnVector stringExpected = ColumnVector.fromStrings("0", "100", null, null, "MIN_VALUE", null)) {
       assertColumnsAreEqual(v0, intResult);
       assertColumnsAreEqual(stringExpected, stringResult);
-      assertColumnsAreEqual(v2, noMaskResult);
+      assertThrows(IllegalStateException.class, () -> v2.mergeAndSetValidity(BinaryOp.BITWISE_AND));
```

Comment:
I am really confused here.  If you don't pass in any children, then there is nothing to change about the validity. Why is this an exception and not a special case copy?